### DISCUSSION TITLE
Agrarwetter Seite erstellen.  Create agricultural weather page.

### DISCUSSION THREAD

**Discussion by OE3MNS** [Daveiano/weewx-wdc#206] [Q&A]
Hallo! 
Erstmal vielen Dank für deine tolle Arbeit mit dem Skin. 
Zu meiner Frage, ich möchte gern ein zusätzliches Agrarwetter Seite erstellen, mit Grafiken von Blattfeuche, Bodentemperatur, Bodenfeuchte, ET, Regen, usw. 
Leider fehlt mir das Wissen, es zu realisieren. Die Grafiken einzufügen per HTML? 
Ist das überhaupt möglich? Und wenn ja, bräuchte ich bitte eine Hilfe. Herzlichen Dank. 
[TEST Link zur meiner Wetterstation](https://wetter-kreuth.at/)
Hello!
First of all, thank you for your great work with the skin.
Regarding my question, I would like to create an additional agricultural weather page, with graphics of leaf moisture, soil temperature, soil moisture, ET, rain, etc.
Unfortunately I don't have the knowledge to realize it. Insert the graphics via HTML?
Is that even possible? And if so, I need some help please. Thank you very much.

**Comment by OE3MNS** [comment]
Problem gelöst! 

Schöne Grüße aus Tulln

**Comment by Daveiano** [comment]
Vielen Dank für die netten Worte!

Ich habe es schon gesehen: https://www.wetter-kreuth.at/agrar.html - Sehr schön!
Wie wurde das realisiert? Ich vermute eine Seite wie `sensor-status.html.tmpl` kopiert und neue Werte für `*_diagram_observations`, etc festgelegt?

Vielleicht sollte ich dazu einen Wiki beitrag schreiben!?

**Reply by OE3MNS** [reply]
Ja, die Monitor Seite hab ich dafür kopiert.
